vllm/core/scheduler.py

```diff
@@ -273,11 +276,19 @@ def __init__(
         BlockSpaceManagerImpl = BlockSpaceManager.get_block_space_manager_class(
             version)
 
+        num_gpu_blocks = cache_config.num_gpu_blocks
```

Comment:
Is this necessary? IIRC, the num_gpus_blocks is per GPU, so there's no need to split it by pp size?

Reply:
This is because we have PP size number of Caches per GPU now.